core/src/main/java/de/danoeh/antennapod/core/util/HTMLUtils.java

```diff
@@ -0,0 +1,29 @@
+package de.danoeh.antennapod.core.util;
+
+import android.text.TextUtils;
+
+import java.util.regex.Pattern;
+
+/**
+ * HTMLUtils contains utility methods to handle HTML content seen in the podcasting world
+ */
+public final class HTMLUtils {
```

Comment:
We follow the Google Java style guide. Please rename to `HtmlUtils`. https://google.github.io/styleguide/javaguide.html#s5.3-camel-case